build-ova.sh

```diff
@@ -19,6 +19,7 @@ function checkout_image_builder_branch() {
     # Check out image builder with specific commit for the
     # corresponding k8s version
     cd ${image_builder_root}
+    git pull
```

Comment:
If the image was built already, docker won't build it again and the latest changes won't be present if the docker image is very old. To avoid these scenarios, pull the code again to make sure we are at TOT.